justfile

```
@@ -78,6 +80,13 @@ run-regtest args="":
     --dart-define="ESPLORA_ENDPOINT={{public_regtest_esplora}}" --dart-define="COORDINATOR_P2P_ENDPOINT={{public_regtest_coordinator}}" \
     --dart-define="COORDINATOR_PORT_HTTP={{public_coordinator_http_port}}"
 
+run-local args="":
```

if it's specific to android, let's call it as such
```suggestion
run-local-android args="":
```